metadata-ingestion/tests/unit/config/write_to_file_directive.yml

```diff
@@ -0,0 +1,11 @@
+foo: bar
+nested:
+  hi: hello
+  __DATAHUB_TO_FILE_ssl_cert: |
```

Comment:
so how does the end user provide this? via the recipe or via environment variable?

Reply:
can be either

`__DATAHUB_TO_FILE_<key>: <value>` will get turned into `<key>: <path to file containing value>`

and value can be specified inline or using an env var/secret